src/go/vtunnel/cmd/host_windows.go

```diff
@@ -56,6 +57,8 @@ and a given port acting as a host end of the tunnel.`,
 
 func init() {
 	hostCmd.Flags().String("config-path", "", "Path to the vtunnel's yaml configuration file")
-	hostCmd.MarkFlagRequired("config-path")
+	if err := hostCmd.MarkFlagRequired("config-path"); err != nil {
+		logrus.WithError(err).Fatal("Failed to set up flags")
```

Comment:
I noticed that you lowered cased the errors in `src/go/nerdctl-stub/main_windows.go` we should probably do the same here.

Reply:
I thought the normal usage is to start error messages with a lowercase letter, but log messages with an uppercase one.

Reply:
This always confuses me, but as long as Linter is happy I'm happy.

Reply:
Log messages are typically full sentences; error messages are wrapped snippets that are concatenated together, so they look odd if they pretend to be sentences and start with an uppercase letter (and/or have a `.` at the end).

At least that's how I understand the reasoning behind these rules.